version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.64-SNAPSHOT"
+version in ThisBuild := "0.0.65-SNAPSHOT"
```

Comment:
Isn't this handled by the Jenkins build. Generates a new PR with version updated